src/main/resources/db/migration/local+dev/R__1_create_dev_users.sql

```diff
@@ -3,10 +3,203 @@
 --These are randomly generated names
 
 INSERT INTO "users" (id, name, delius_username, delius_staff_identifier, probation_region_id, delius_staff_code) VALUES
-    ('aa30f20a-84e3-4baa-bef0-3c9bd51879ad', 'Default User', 'JimSnowLdap', 2500041001, '43606be0-9836-441d-9bc1-5586de9ac931', 'STAFFCODE'), -- South West
+    ('aa30f20a-84e3-4baa-bef0-3c9bd51879ad', 'Default User', 'JIMSNOWLDAP', 2500041001, '43606be0-9836-441d-9bc1-5586de9ac931', 'STAFFCODE'), -- South West
     ('f29b6402-b5f0-4f39-a66f-a2bb13eb5d6d', 'A. E2etester', 'APPROVEDPREMISESTESTUSER', 2500041002, 'ca979718-b15d-4318-9944-69aaff281cad', 'STAFFCODE'), -- East of England
     ('0621c5b0-0028-40b7-87fb-53ec65704314', 'T. Tester', 'temporary-accommodation-e2e-tester', 2500041003, 'afee0696-8df3-4d9f-9d0c-268f17772e2c', 'STAFFCODE'), -- Wales
     ('b5825da0-1553-4398-90ac-6a8e0c8a4cae', 'Tester Testy', 'tester.testy', 2500043547, 'c5acff6c-d0d2-4b89-9f4d-89a15cfa3891', 'STAFFCODE'), -- North East
     ('695ba399-c407-4b66-aafc-e8835d72b8a7', 'Bernard Beaks', 'bernard.beaks', 2500057096, 'ca979718-b15d-4318-9944-69aaff281cad', 'STAFFCODE'), -- East of England
     ('045b71d3-9845-49b3-a79b-c7799a6bc7bc', 'Panesar Jaspal', 'panesar.jaspal', 2500054544, 'afee0696-8df3-4d9f-9d0c-268f17772e2c', 'STAFFCODE') -- Wales
 ON CONFLICT (id) DO NOTHING;
+
+UPDATE
+  users
+SET
+  "delius_username" = 'JIMSNOWLDAP'
+WHERE
+  id = 'aa30f20a-84e3-4baa-bef0-3c9bd51879ad';
+
+INSERT INTO
+  "user_role_assignments" ("id", "role", "user_id")
+VALUES
+  (
+    '4d6500ff-0670-4a8e-8581-dff45292b2e4',
+    'ASSESSOR',
+    'aa30f20a-84e3-4baa-bef0-3c9bd51879ad'
+  ),
+  (
+    'f78a5a6d-7d15-415c-b10f-91c0dff16753',
+    'MATCHER',
+    'aa30f20a-84e3-4baa-bef0-3c9bd51879ad'
+  ),
+  (
+    'b7f476af-0e7e-44ba-9c66-abe27d8c70ba',
+    'MANAGER',
+    'aa30f20a-84e3-4baa-bef0-3c9bd51879ad'
+  ),
+  (
+    '0354ff61-295a-4d06-b9e7-ae9fd53eba12',
+    'WORKFLOW_MANAGER',
+    'aa30f20a-84e3-4baa-bef0-3c9bd51879ad'
+  ),
+  (
+    'dd949e2b-5f9b-4560-b3bc-c3ad6793fb28',
+    'APPLICANT',
```

Comment:
`APPLICANT` is not a role, so will cause the enum to blow up...

Reply:
Is it not? I was going from UserRoles as per https://github.com/ministryofjustice/hmpps-approved-premises-api/blob/4928a7455cbb35274d19adbd0563c24301a1f230/src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/UserEntity.kt#L79

Reply:
Interesting! I stand corrected -- it won't blow up as the `APPLICANT` enum is indeed present. 

However, APPLICANT is not an internally managed role. It's a pseudo-role. It refers to the capabilities which a logged in user defaults to when they don't have any of the internally managed roles. i.e as an applicant you can make and view applications for people who are managed by teams of which you are a member...

So I think it ought to be removed from the enum...?